en_US.json

```diff
@@ -111,6 +111,8 @@
 	"button.text.previous": "Previous",
 	"button.title.next": "Next",
 	"button.upgrade.title": "Upgrade",
+	"cacertificate.create.title":"Create a CA Certificate",
```

Comment:
For the "Create XXXX" buttons on portal, we do not use article elsewhere. It might make sense to have the article, but let's keep consistency across the pages.
Let's use "Create CA Certificate" here.

Reply:
Updated the text.

Reply:
OK. Didn't notice the nuance. Thanks for the information.